dotnet/src/Microsoft.AutoGen/Abstractions/Microsoft.AutoGen.Abstractions.csproj

```diff
@@ -20,7 +20,13 @@
     <PackageReference Include="Grpc.AspNetCore" />
     <PackageReference Include="Grpc.Net.ClientFactory" />
     <PackageReference Include="Grpc.Tools" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Orleans.Core.Abstractions" />
+ 		<PackageReference Include="Microsoft.Orleans.Sdk" />
 	  <PackageReference Include="Microsoft.SemanticKernel" />
   </ItemGroup>
+  <PropertyGroup>
```

Comment:
Why we set `ServerGarbageCollection` and `ConcurrentGarbageCollection` in abstraction package

Reply:
trying to force Orleans to stop warning about that. @ReubenBond what is the right way to do this?

Reply:
removed for now since it wasn't doing what I wanted anyway